src/models/regression/index.ts

```diff
@@ -1,6 +1,10 @@
+import fs from "fs";
```

Comment:
Don't use `fs`. the libs you use should be browser friendly. `fs` may not run on browser. it's a nodejs lib.

So use a different lib.